src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -55,9 +59,15 @@ const SubscriberHistory = ({ location, dependencies: { dopplerApiClient } }) =>
         if (!response.success) {
           setState({ loading: false });
         } else {
+          const sentCampaigns = response.value.items.map((campaign) => {
+            return {
+              ...campaign,
+              imgPreview: searchLinkByRel(campaign.links, '/docs/rels/get-campaign-preview'),
+            };
```

Comment:
In my opinion, this mapping work is a responsibility of _dopplerApiClient_.

Reply:
i moved now to the service!